hazelcast/src/test/java/com/hazelcast/test/annotation/IntegrationTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+/**
+ * Tests which should be executed after a final JAR is packaged.
+ * These tests are executed by the Fail-Safe Maven plugin.
+ * (other tests are executed by the Surefire plugin)
+ *
+ */
+public final class IntegrationTest {
```

Comment:
I'm not sure why we need this. Isn't the `IT` suffix enough for filtering the tests for the failsafe plugin?